### ISSUE TITLE
[BUG] XCH lost

### ISSUE THREAD

**Issue by so-y** [Chia-Network/chia-blockchain#3660]
I receive 0.5 XCH but I cannot see it in my wallet (both the GUI and the CLI). This is my address. (https://www.chiaexplorer.com/blockchain/address/xch1zmecs9zmyl4vackvfg3xluq5pzz08r6r434z6c99hu2wfewat3rqunvfmw)

How to retrieve my XCH?

```
Wallet height: 229480
Sync status: Synced
Balances, fingerprint: 127xxxx018
Wallet ID 1 type STANDARD_WALLET
   -Total Balance: 0.0 xch (0 mojo)
   -Pending Total Balance: 0.0 xch (0 mojo)
   -Spendable: 0.0 xch (0 mojo)
```

![image](https://user-images.githubusercontent.com/25190781/116999147-ce2c6f80-ad11-11eb-8eab-33156cdcc687.png)

**Comment by ToughBODY** [comment]
mee too

**Comment by ToughBODY** [comment]
I receive 2 XCH but I cannot see it in my wallet

**Comment by mariano54** [comment]
Are you sure you are logged in to the correct key? Make sure that they key you are logged in to, is the one which you used to get the address.

**Comment by DIlbert25** [comment]
> I receive 0.5 XCH but I cannot see it in my wallet (both the GUI and the CLI). This is my address. (https://www.chiaexplorer.com/blockchain/address/xch1zmecs9zmyl4vackvfg3xluq5pzz08r6r434z6c99hu2wfewat3rqunvfmw)
> 
> How to retrieve my XCH?
> 
> ```
> Wallet height: 229480
> Sync status: Synced
> Balances, fingerprint: 127xxxx018
> Wallet ID 1 type STANDARD_WALLET
>    -Total Balance: 0.0 xch (0 mojo)
>    -Pending Total Balance: 0.0 xch (0 mojo)
>    -Spendable: 0.0 xch (0 mojo)
> ```
> 
> ![image](https://user-images.githubusercontent.com/25190781/116999147-ce2c6f80-ad11-11eb-8eab-33156cdcc687.png)

Me too.

I received 02 XCH.

I did the update to 1.1.3, after that, my wallet and my full node DON´T sync.

Anything wrong.

**Comment by dorkmo** [comment]
you might try installing on a different computer if you have one available

**Comment by so-y** [comment]
Sorry, I found the reason. To reduce the syncing time, I reuse another database (another private key). However, the database of the wallet cannot be reused because the algorithm might traverse all the blocks from zero. I cannot see the transaction because my wallet thinks it has searched the blocks before mistakenly. After removing the database of the wallet. I found my XCH. Thanks.

**Comment by PAVoutsinas** [comment]
I had a similar issue that i was stuck on. Deleting the db folder in the wallet directory worked. Although  to get the chia app to resync wallet I also had to use the gui interface, "delete all keys" option when it first opens, then re-enter the 24 phrase passcode. Takes like a day to resync wallet. This procedure will only resync the wallet. Farm will stay synced. My issue was caused by running full nodes on more than one computer.
